src/index.html

```diff
@@ -10,13 +10,70 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a
+          href="#"
+          class="logo"
+        >
+          <img
+            src="images/logo.png"
+            alt="Moyo"
+            class="logo_img"
+          />
+        </a>
+
+        <nav class="nav">
+          <ul class="links">
+            <li class="link">
+              <a
+                class="is-active"
```

Comment:
Fix classes everywhere
```suggestion
          <ul class="nav__list">
            <li class="nav__item">
              <a
                class="nav__link is-active"
```